example/assets/js/App.tsx

```diff
@@ -19,22 +29,69 @@ const theme = createTheme({});
 configAIAssistant({ BASE: "ai-assistant" });
 
 const ExampleIndex = () => {
+  const [showAlert, setShowAlert] = React.useState<boolean>(true);
```

Comment:
Can we only show that if the user is logged out?

Reply:
Hm, a workaround would be to do a simple request to the API, using an existing endpoint, and then check if the response status is 401. Would that work for the examples? I can add a code comment saying that in real projects it should do a request related to the User entity.

Reply:
Yes, that's fine